coordinator/points_writer.go

```diff
@@ -121,32 +124,18 @@ func (s *ShardMapping) MapPoint(shardInfo *meta.ShardInfo, p models.Point) {
 
 // Open opens the communication channel with the point writer.
 func (w *PointsWriter) Open() error {
-	w.mu.Lock()
-	defer w.mu.Unlock()
```

Comment:
Was this causing problems, or is this cleanup of unneeded code?

Reply:
Cleanup of unneeded code